site2/docs/security-oauth2.md

```diff
@@ -195,6 +195,20 @@ This example shows how to configure OAuth2 authentication in Node.js client.
 ```
 > Note: The support for OAuth2 authentication is only available in Node.js client 1.6.2 and later versions.
 
+## Broker configuration
+To enable OAuth2 brokers authentication in brokers, add the following parameters to the `broker.conf` or `standalone.conf` file.
```

Comment:
`brokers` can be removed between `OAuth2` and `authentication`?

Reply:
yes, I think so, and this is a suggestion given by your. 
My fault!  I did not review it carefully.